client/components/Overview/components/Sizes.jsx

```diff
@@ -2,6 +2,7 @@
 /* eslint-disable no-restricted-syntax */
 import React from 'react';
 import SizesRender from './SizesRender';
+import '../styles/SizesStyles.css';
 
 //
 const Sizes = ({
```

Comment:
... is this another way to declare a function without using onst funcName = () => {} ?

Reply:
This is just how I am importing my css. My css doesn't have any functions that I need to call in my code. Since the ids being reference are in my css file doing it this way imports those style changes to the respective ids.